client/app/bundles/course/survey/pages/ResponseIndex/index.jsx

```diff
@@ -243,8 +243,8 @@ class ResponseIndex extends React.Component {
           </TableBody>
         </Table>
         <CardText>
-          <RemindButton />
```

Comment:
RemindButton's ConfirmationDialog is causing everything to its right to be pushed to the next line, even when it's hidden.